tox.ini

```diff
@@ -17,6 +17,7 @@ commands =
     # This repository uses a Markdown long_description, so the -r flag to
     # `setup.py check` is not needed. If your project contains a README.rst,
     # use `python setup.py check -m -r -s` instead.
+    pip install -e . --install-option="--antlr-runtime={env:ANTLR4_RUNTIME_TYPE:static}" --no-use-pep517
     python setup.py develop
```

Comment:
I believe the `setup.py develop` isn't needed (implied by the `-e) and the check isn't needed either?

Reply:
Also, if the `-e` is breaking things, I think we can drop it IMO.

Reply:
I've removed these lines. `-e` works fine.